src/rpc/blockchain.cpp

```diff
@@ -130,9 +130,15 @@ UniValue blockToJSON(const CBlock& block, const CBlockIndex* blockindex, bool tx
     result.pushKV("size", (int)::GetSerializeSize(block, SER_NETWORK, PROTOCOL_VERSION));
     result.pushKV("weight", (int)::GetBlockWeight(block));
     result.pushKV("height", blockindex->nHeight);
-    result.pushKV("proof_type", blockindex->IsProofOfStake() ? "Proof-of-Stake" : "Proof-of-Work");
+    result.pushKV("proof_type", blockindex->IsProofOfStake() ? "Proof-of-Stake" : blockindex->IsProgProofOfWork() ? "Proof-of-work (ProgPow)" : "Proof-of-Work");
```

Comment:
Include outputs for  "Proof-of-work (RandomX)" and "Proof-of-work (Sha256D)"

Reply:
Done